.github/ISSUE_TEMPLATE/docs_issue.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Docs Issue
+about: Report issues or request new content in the Crossplane documentation.
+labels: ["docs", "exempt-from-stale"]
```

Comment:
can you share a bit more of your thinking for why all docs bugs should get the `exempt-from-stale` label?

Reply:
Doc issues should be managed in triage. It's either a bug that should be quickly fixed or a content request that can be put on a roadmap. If it's neither of these then it should be closed/won't fix.

Reply:
> Doc issues should be managed in triage. It's either a bug that should be quickly fixed or a content request that can be put on a roadmap. If it's neither of these then it should be closed/won't fix.

I'd prefer to avoid making any issue class exempt from stale.

A large part of the stale bot existing is because the Crossplane maintainers have been unable to keep up with manually grooming the issues raised by the community. It sounds like you're optimistic that we (you?) will be able to do that for docs issues. If that is indeed the case then whether the stale bot is enabled or not for docs issues shouldn't matter; they'll never get a chance to be marked stale because someone will triage and address them well before the 90 days of inactivity that would cause them to be marked stale.

Reply:
I removed the `exempt-from-stale` label.